src/parser/view_transform.c

```diff
@@ -3162,6 +3207,68 @@ pt_copypush_terms (PARSER_CONTEXT * parser, PT_NODE * spec, PT_NODE * query, PT_
   return;
 }
 
+/*
+ * mq_copypush_sargable_terms_dblink() -
+ *   return:
+ *   parser(in):
+ *   statement(in):
+ *   spec(in):
+ *   new_query(in/out):
+ *   infop(in):
+ */
+static int
+mq_copypush_sargable_terms_dblink (PARSER_CONTEXT * parser, PT_NODE * statement, PT_NODE * spec, PT_NODE * new_query,
+				   FIND_ID_INFO * infop)
+{
+  PT_NODE *term, *new_term, *push_term_list;
+  int push_cnt, copy_cnt;
+  PT_NODE *save_next;
+
+  /* init */
+  push_term_list = NULL;
+  push_cnt = 0;
+  copy_cnt = -1;
+
+  for (term = statement->info.query.q.select.where; term; term = term->next)
+    {
+      if (pt_sargable_term (parser, term, infop) && PT_PUSHABLE_TERM (infop))
```

Comment:
pt_sargable_term() should also be separated. In dblink, only the attr of the same spec should be checked.

Reply:
When I checked it again, it seems that it is okay to use pt_sargable_term().
I recommend that various tests will be performed with this routine.

Reply:
Ok, I agree. Thanks.